### DISCUSSION TITLE
Turborepo monorepo support

### DISCUSSION THREAD

**Discussion by jt561** [resend/react-email#724] [Ideas]
### Goals

Hello, first things first, you guys are doing great work!

I was wondering if there was a good way/example of how to integrate react-email in a mono repo like [Turborepo](https://turbo.build/repo)?

And if there is a recommended way, then maybe we could provide a demo project as this would be really helpful for people using mono repos with react-email in the future.

### Background

I am currently working on a project that uses Turborepo](https://turbo.build/repo) mono repo and would like to add react-email as a workspace in that project but I seem to be running into issues. 

I have run react-email in a simple repo and everything works great but not so well in the mono repo.

One of the issues with the mono repo implementation is that shutting down the mono repo 'dev' command does not kill the node environment that runs react-email so I need to go into my Task manager to kill it.
Another issue is that the 'email dev' command doesn't work quite well in mono repos as react-email creates another folder inside the mono repo which breaks a few things.

### Proposal

Recommend a way to work with mono repos

**Comment by jt561** [comment]
So I have noticed that the issue has to do with installing dependencies. Ideally, I don't want to add the **.react-email** folder as a workspace in my mono repo but that means the dependencies for it do not get installed which causes import errors.

**Comment by maxprilutskiy** [comment]
+1 here, the steps described in the readme don't work when I'm trying to use react-email project as monorepo package. That's unfortunate.

**Comment by uncvrd** [comment]
Agreed here - adding to this, once the `.react-email` folder is created in a workspace, I'd like to run `turbo run dev`. This looks for `package.json`'s with a `dev` script....which means that both `react-email`'s `dev` script will be called AND the `.react-email` NextJS `dev` script will be called (this shouldn't be called since the `email dev` takes care of calling the `.react-email` `dev` script).

I think this is where my problems are

**Comment by maxprilutskiy** [comment]
For me it's not only that ^^, but also the fact that dependencies don't get installed properly when using react-email inside a monorepo. Given monorepo is de-facto the standard in many orgs nowadays, it makes react-email simply unusable. ☹️ ☹️

**Comment by chihabhajji** [comment]
obligatory +1

**Comment by gabrielmfern** [comment]
We have actually implemented a way so that the `email dev` command has proper support for all kinds of monorepos. It should be working ever since `react-email@2.0.0`.
